Dockerfile

```
@@ -0,0 +1,17 @@
+###############################################
+# Dockerfile to build Stranger container image
+#   docker build -t gvcn/stranger:v0.8.1 . --platform linux/amd64
+#   docker push gvcn/stranger:v0.8.1
+#   docker run -v $PWD:$PWD gvcn/stranger:v0.8.1 stranger --help
+#   docker run -v $PWD:$PWD gvcn/stranger:v0.8.1 stranger -f /bin/stranger/stranger/resources/variant_catalog_hg38.json $PWD/sample.joint.repeats.merged.vcf.gz > $PWD/sample.repeats.stranger.vcf
+###############################################
+FROM continuumio/miniconda3:23.3.1-0-alpine
+
+# Install git
+RUN conda install -c anaconda git && \
+    cd /bin && \
+    git clone --branch v0.8.1 --depth 1 https://github.com/Clinical-Genomics/stranger.git && \
```

I would suggest keeping the versioning by repo- and dockerhub tags rather than editing docker files!
```suggestion
    git clone --depth 1 https://github.com/Clinical-Genomics/stranger.git && \
```

---

Also not sure why we don't just go with the currently checked out version, but let's just merge this for you and I'll test an alternative that I hope we can both be happy with!